.github/workflows/CD.yml

```diff
@@ -36,9 +37,10 @@ jobs:
       # IMPORTANT: this permission is mandatory for trusted publishing
       id-token: write
     steps:
-      - uses: actions/download-artifact@v3
+      - uses: actions/download-artifact@v4
         with:
-          path: artifact/
+          name: release
+          path: dist
```

Comment:
I noticed these didn't match, but just assumed this was correct. I guess it wasn't?

Reply:
Yeah. The strange thing is that I didn't touch these...

Reply:
Best guess is it had to do with the version change of the pypi-publish action